P/PAPI/build_tarballs.jl

```diff
@@ -63,16 +63,19 @@ make -j ${nproc}
 make install
 """
 
-# These are the platforms we will build for by default, unless further
-# platforms are passed in on the command line
+# These are the platforms we will build for by default
 platforms = [
```

Comment:
That's all Linux platforms, isn't it?

Reply:
I changed the comment because the logic below only ever installs platforms listed here, it won't install "further platforms". Usually you can pass an unlisted platform to build, but not with this script.

Reply:
Yes, I think these are all the Linux platforms there are.

Reply:
My point was that this can be simplified to something like supported_platforms(;exclude=!Sys.islinux)

Reply:
Done.